10.0/alpine-fpm/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM php:7.0-fpm-alpine
+
+# set recommended PHP.ini settings
+# see https://secure.php.net/manual/en/opcache.installation.php
+RUN { \
+    echo 'opcache.memory_consumption=128'; \
+    echo 'opcache.interned_strings_buffer=8'; \
+    echo 'opcache.max_accelerated_files=4000'; \
+    echo 'opcache.revalidate_freq=60'; \
+    echo 'opcache.fast_shutdown=1'; \
+    echo 'opcache.enable_cli=1'; \
+  } > /usr/local/etc/php/conf.d/opcache-recommended.ini
+
+# Install PHP extensions
+# https://docs.nextcloud.com/server/9/admin_manual/installation/source_installation.html
+RUN set -ex \
+  && apk update \
+  && apk add build-base python-dev py-pip jpeg-dev jpeg \
+     libldap openldap-dev zlib zlib-dev \
+     freetype freetype-dev libjpeg-turbo libjpeg-turbo-dev \
+     postgresql-dev libmcrypt-dev libmcrypt libpng-dev libpng \
+     autoconf make g++ gcc git file gnupg re2c icu icu-dev tzdata \
+  && docker-php-ext-configure gd \
+     --with-freetype-dir=/usr/include/ \
+     --with-png-dir=/usr/include \
+     --with-jpeg-dir=/usr/include \
+  && NPROC=$(getconf _NPROCESSORS_ONLN) \
+  && docker-php-ext-install -j${NPROC} gd \
+  && docker-php-ext-install exif intl mbstring mcrypt opcache \
+     pdo_mysql pdo_pgsql pgsql zip ldap \
+  && docker-php-ext-enable gd exif intl mbstring mcrypt opcache \
+     pdo_mysql pdo_pgsql pgsql zip ldap \
+  && pecl install APCu-5.1.6 \
+  && git clone https://github.com/phpredis/phpredis.git \
+  && cd phpredis \
+  && git checkout php7 \
+  && phpize \
+  && ./configure \
+  && make && make install \
```

Comment:
There were problems with php7 and Redis when I made the first version of the image but I'll try again. It would be cleaner.
Also, Memcache for php7 was broken then.